server/routes/sightingsController.js

```diff
@@ -0,0 +1,38 @@
+// const mongoose = require('mongoose');
+
+const Sighting = require('../models/Sighting.js');
+
+// Get all sightings in the database
+exports.getSightings = function (req, res) {
+	console.log('Getting Sightings...');
+	Sighting.find({}).then((sight) => {
```

Comment:
Should add optional params for sorting/filtering/pagination now, rather than try to refactor them in later